### PR TITLE
Add local crash tracking and viewing

### PR DESCRIPTION
**Pull request by camporter**
Uses https://github.com/FunkyMuse/Crashy to save and access previous app crashes.

https://github.com/dessalines/jerboa/assets/494446/82a447cc-f972-4104-80a2-4a9ac7ba3bba

### CODE REVIEW

**Comment by camporter** [comment]
I've left this in draft just to gauge the feelings behind local crash reporting. There was an issue discussing this and other solutions, but I can't seem to find it now.

**Comment by MV-GH** [comment]
I think you mean #681. This pretty great but I'm guessing if you crash at startup this won't be useable?

**Comment by camporter** [comment]
Right. Start up crashes should still be recorded depending on how bad they are, but won't be accessible to a user without doing something to avoid the crash.

This avoids transmitting crashes somewhere that users might not like or trust, and is at least better than no way (besides system crash reporting) to get crashes to report.

**Review by dessalines — COMMENTED** [review]
Seems like a good idea to have this, thanks. Fix conflicts then we can test.

**Comment by camporter** [comment]
:+1: rebased

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -102,6 +102,10 @@
     <string name="community_sortBy">Sort by</string>
     <string name="community_subscribe">Subscribe</string>
     <string name="community_users_month">%1$s users / month</string>
+    <string name="crashes">Crashes</string>
```
Comment (twizmwazin) [inline app/src/main/res/values/strings.xml:105]: Maybe "Crash logs", "All crash logs deleted", "Copy crash log" and "Delete all crash logs"?

Comment (camporter) [reply app/src/main/res/values/strings.xml]: Yes that's much easier to read, changed 👍

**Review by dessalines — APPROVED** [review]
Tested this, and it seems to work. Thx!

**Review by MV-GH — APPROVED** [review]

**Pull request merged**